src/jit/codegenarm64.cpp

```diff
@@ -2616,9 +2616,15 @@ CodeGen::genCodeForTreeNode(GenTreePtr treeNode)
                 if (targetType == TYP_STRUCT)
                 {
                     // At this point any TYP_STRUCT LclVar must be a two register argument
-                    assert(varDsc->lvSize() == 16); 
-                    emit->emitIns_R_S(ins_Load(TYP_I_IMPL), emitTypeSize(TYP_I_IMPL), targetReg, varNum, 0);
-                    emit->emitIns_R_S(ins_Load(TYP_I_IMPL), emitTypeSize(TYP_I_IMPL), REG_NEXT(targetReg), varNum, TARGET_POINTER_SIZE);
+                    assert(varDsc->lvSize() == 2*TARGET_POINTER_SIZE);
```

Comment:
I really like to see an implementation design for multi-reg structs for arm64 that provides details on how we intend to model the operations in IR, register requirements specified to LSRA and what is required from LSRA side.

IMO, this feels like short term work.  I am fine with it if you think it will help in final implementation.